test/util.js

```diff
@@ -0,0 +1,21 @@
+import test from 'ava';
+import {Observable as RxObservable} from 'rxjs/Observable';
+import ZenObservable from 'zen-observable';
+import Listr from '../';
+import {isListr, isObservable} from '../lib/util';
+
+test('isListr', t => {
+	t.falsy(isListr(null));
```

Comment:
Initially I did. But it failed, since right now `isListr()` will return `null` or the listr instance itself if it is. So then I just added a `... ? true : false` to `isListr()`, but `xo` failed, because for some weird reason it complains that the ternary is superfluous...

Any suggestions?